assets/sass/v2/_phone-challenge.scss

```diff
@@ -1,6 +1,10 @@
 .phone-authenticator-challenge {
   .okta-form-subtitle {
     text-align: center;
+
+    p {
+      margin-top: 1em;
```

Comment:
`em` is relative to the font-size of the element in question (older way). A better unit is `rem` which is the font-size of body and it's the newer way of doing the same. I'd suggest to change to `rem`.

Reply:
sounds good